testing/benchmark/system-profiles/32GBx2zone.tfvars

```diff
@@ -3,10 +3,14 @@ user_name = "USER"
 # The number of AZs the APM Server should span.
 apm_server_zone_count = 1
 # The Elasticsearch cluster node size.
-elasticsearch_size = "128g"
+elasticsearch_size = "256g"
 # The number of AZs the Elasticsearch cluster should have.
 elasticsearch_zone_count = 2
+# Run the cluster with a dedicated master
+elasticsearch_dedicated_masters = true
 # APM server instance size
 apm_server_size = "32g"
+# Number of shards for the ES indices
+apm_shards = 4
```

Comment:
Is there a specific reason this wasn't set to 8? (1 per 64GB => 256: 4x2)

Reply:
The benchmark results are the same whether we go to 4 or 8 shards for 32GB, or 2 to 4 with 16GB.